Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM ruby:2.6.6
+ARG ruby_version
+
+FROM ruby:${ruby_version}
```

Comment:
If we modify this we should also modify `script/build_images.sh` otherwise docker hub won't have the images published because the script will crash no?

Reply:
I've copied the original Dockerfiles to a `dockerhub/` directory and updated circleci build to use them, so publishing to Dockerhub continues smoothly until we finish the migration to Github Actions. See https://github.com/decidim/docker/pull/50/commits/14559839d3ee50f62aff6239644ba5d5d44c9001